modules/installation-vsphere-infrastructure.adoc

```diff
@@ -6,21 +6,25 @@
 [id="installation-vsphere-infrastructure_{context}"]
 = VMware vSphere infrastructure requirements
 
-You must install the {product-title} cluster on a VMware vSphere version 6.5
-or 6.7U2 or later instance.
+You must install the {product-title} cluster on a VMware vSphere version that meets the requirements for the components that you use.
 
-VMware recommends using vSphere Version 6.7 U2 or later with your {product-title}
-cluster.
-vSphere 6.7U2 includes:
+.Minimum supported vSphere version for VMware components
+|===
+|Component | Minimum supported versions |Description
 
-* Support for VMware NSX-T
-* Support for vSAN, VMFS and NFS, using the in-tree VCP
+|Hypervisor
+|vSphere 6.5 with HW version 13
+|This version is the is the minimum version that {op-system-first} supports. See the link:https://access.redhat.com/ecosystem/search/#/ecosystem/Red%20Hat%20Enterprise%20Linux?sort=sortTitle%20asc&vendors=VMware&category=Server[Red Hat Enterprise Linux 8 supported hypervisors list].
 
-While vSphere 6.5 with Hardware version 13 is supported, {product-title}
-clusters are subject to the following restrictions:
+|Networking (NSX-T)
+|vSphere 6.5U3 or vSphere 6.7U2+
+|Previous versions of vSphere are not supported.
```

Comment:
I'm not quite sure if NSX-T is a required component to list here, and from description in bz1803979, ocp does not support it currently.

Reply:
@dav1x 
QE's vcsa1 is 6.5.0.23000, and vcsa2 is 6.7.0.20000, not quite sure about U* version, do u know how to check the U vesion?

Reply:
@katherinedube, will you PTAL at this description of NSX-T for me?

Reply:
It's correct. This information comes right from the VMware PM team and what they're supporting in their product (happy to forward original email thread.) Otherwise, the BZ text is very mis-leading and just creating confusion. My only suggestion would be to clean up the text to say: "VMware vSphere 6.5U3 or 6.7U2+ is required. Previous versions of vSphere are not supported with NSX-T." Also, please have Maria Bracho (PM for VMware install) and Duncan Hardie (PM for storage) review this PR. Thanks!